### PR TITLE
Initial content

### PR DESCRIPTION
**Pull request by tebeka**
Initial content with many sections removed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,506 +1,534 @@
...
+	    <div class="row">
+		<div class="col-md-5 col-md-offset-1">
+		    <div class="about_text">
+			<h3 class="wow slideInLeft" data-wow-duration="1s">About Us</h3>
+			<h2 class="wow slideInLeft" data-wow-duration="1.3s">GopherCon is focused on<br>the Go programming
+			    language</h2>
+			<p class="wow slideInLeft" data-wow-duration="1.8s">
+			GopherCon Israel 2019 is the first GopherCon to be held in
+			Israel. The conference will be held on February 11 at the
+			Tel-Aviv convention center. 250 Gophers will attend the
```
Comment (bivas) [inline index.html]: Tel Aviv **C**onvention **C**enter

**Review by bivas — CHANGES_REQUESTED** [review]
1. There are additional links in the page that requires update
2. Don't comment out sections, please add `hidden-md` and `hidden-sm` as class attributes (this should hide the section
3. What about Google Analytics?

File: index.html (hunk 1 above)
Comment (tebeka) [reply index.html]: > There are additional links in the page that requires update

And they are?

Comment (tebeka) [reply index.html]: > Don't comment out sections, please add hidden-md and hidden-sm as class attributes (this should hide the section

Why is it better than a comment?

Comment (tebeka) [reply index.html]: > What about Google Analytics?

Will add.

Comment (bivas) [reply index.html]: At the footer there are facebook, twitter, code of conduct etc links

File: index.html [hunk 2]
```diff
@@ -1,506 +1,546 @@
...
+			    <li><a href="#" title="Speakers">Speakers</a></li>
+			    -->
+			    <li><a href="#" title="Social Area">Social Area</a></li>
+			</ul>
+		    </div>
+		</div>
+		<div class="col-md-4 col-sm-4 col-xs-12">
+		    <div class="same_style footersocial">
+			<h6>Code Of Conduct</h6>
+			<p>We operate a <a href="https://golang.org/conduct">code of conduct</a> to ensure all attendees are able to enjoy the conference.</p>
```
Comment (bivas) [inline index.html]: `target="_blank"`

File: index.html [hunk 3]
```diff
@@ -1,506 +1,546 @@
...
+		    <div class="item shake1"></div>
+		</div>
+	    </div>
+	</div>
+    </section>
+    <footer class="footer">
+	<div class="container">
+	    <div class="row">
+		<div class="col-md-5 col-sm-5">
+		    <div class="footerlogo"><a href="index.html"><img alt="footerlogo" src="images/logo.png" title="Gopher Con Israel"></a>
```
Comment (bivas) [inline index.html]: GopherCon (remove space)

**Review by bivas — DISMISSED** [review]

File: index.html (hunk 3 above)
Comment (tebeka) [reply index.html]: Done

File: index.html (hunk 1 above)
Comment (tebeka) [reply index.html]: Done

**Pull request merged**